docs/AVM_Specification.md

```diff
@@ -231,3 +231,5 @@ The instructions are as follows:
 | 0x72                                            | inbox         | Pop a Value (A) off of the Data Stack. Block until either (a) the VM's inbox is non-empty, or (b) the lower bound on current time (as would be returned by the gettime instruction) is greater than or equal to A. Then push the Inbox contents, as supplied by the Runtime Environment, onto the Data Stack, and set the inbox to None.                           | 40          |
 | 0x73                                            | error         | Raise an Error.                                                                                                                                                                                                                                                                                                                                                    | 5           |
 | 0x74                                            | halt          | Enter the Halted state.                                                                                                                                                                                                                                                                                                                                            | 10          |
+| 80s: Precompile Operations                      |               | &nbsp;                                                                                                                                                                                                                                                                                                                                                             |
+| 0x80                                            | ecrecover     | Pop four values off the Data Stack. If not all of the values are integers raise an error. The first two values first and second half of a compact ecdsa signature, the third is the recovery ID of the signature, and the forth is the message. If the recovered ethereum address was valid push the address, otherwise push 0.                                    | 20000       |
```

Comment:
Should say here that this is compatible with [insert name of well-defined EVM precompile].  This spec is supposed to be reasonably clear about the semantics of the instruction, not just the formats and purposes of the args.

Reply:
I added an additional comment here